src/Compilers/Core/Portable/Symbols/IParameterSymbol.cs

```diff
@@ -40,6 +40,8 @@ public interface IParameterSymbol : ISymbol
         /// </summary>
         ITypeSymbol Type { get; }
 
+        Nullability TypeNullability { get; }
```

Comment:
I don't think "Type" belongs in the name.

Reply:
Same response as before. I put it in to give users a key for what the nullability is applying to.

---
In reply to: [205846691](https://github.com/dotnet/roslyn/pull/28860#discussion_r205846691) [](ancestors = 205846691)